camkes-linux-artifacts/camkes-linux-modules/camkes-connector-modules/connection/connection.c

```diff
@@ -87,21 +86,7 @@ static int connector_pci_probe(struct pci_dev *dev,
         goto disable_pci;
     }
 
-    /* Set up the event bar - We assume the event bar is always BAR0
-     * even if the PCI device does not use events */
-    uio->mem[0].addr = pci_resource_start(dev, 0);
-    if (!uio->mem[0].addr) {
-        goto disable_pci;
-    }
-    uio->mem[0].size = pci_resource_len(dev, 0);
-    uio->mem[0].internal_addr = pci_ioremap_bar(dev, 0);
-    if (!uio->mem[0].internal_addr) {
-        goto disable_pci;
-    }
-    uio->mem[0].memtype = UIO_MEM_PHYS;
-    printk("Event Bar (dev-%d) initalised\n", current_free_dev);
-
-    for (i = 1; i < MAX_UIO_MAPS; i++) {
+    for (i = 0; i < MAX_UIO_MAPS; i++) {
```

Comment:
more for the refactoring commit: have `i` just for the loop here
```suggestion
    for (int i = 0; i < MAX_UIO_MAPS; i++) {
```

Reply:
Do we have some minimum gcc version required for seL4? You see, this is Linux kernel module code. I know Linus said Linux 5.18 will move to C11 from C89, along with gcc minimum version required changing to 5.1. The kernels in seL4 examples might even be 4.x series, and I think on those kernels there's no `-std=c11` flag and compiling c99 stuff would fail.

Reply:
So maybe we won't be able to do this until all VM examples use kernel >= 5.18.